arch/arm/soc/nordic_nrf/nrf52/Kconfig.soc

```diff
@@ -44,6 +44,7 @@ config SOC_NRF52810
 
 config SOC_NRF52832
 	depends on SOC_SERIES_NRF52X
+	select CPU_HAS_FPU
```

Comment:
This addition is no longer needed (see 2 lines below).

Reply:
thanks.